tests/unit/conftest.py

```diff
@@ -164,6 +164,37 @@ def start_cmd_handler(argv: list[str]) -> synapse.ExecResult:
     harness.cleanup()
 
 
+@pytest.fixture(name="harness_with_postgresql")
+def harness_with_postgresql_fixture() -> typing.Generator[Harness, None, None]:
```

Comment:
When naming fixtures it would be good to focus on specifically what is provided, e.g., in this case `postgresql`. The fact that it uses the harness seems to be an implementation detail. For fixtures it is also usually recommended to yield something specifically related to the fixture, e.g., something about `postgresql` in this case. The `harness` can always be retrieved from the original fixture

Reply:
Please also consider this for the other fixtures here

Reply:
Thanks for the tip. I chose this naming because this fixture was independent from the base `harness` fixture. I have reworked this fixture to add postgres on top of the base fixture, moved it to `test_database.py` which is the only place where postgres is tested, and also set it to `autouse`.